### PR TITLE
feat: login page

### PR DESCRIPTION
**Pull request by femacedogomes**
Login Page screen with no API integration. Still need's to improve the mobile screen version.

https://www.notion.so/028386dd8cc04ffeaa06931d912bd0fc?v=b8d42189aed746d8a0a716c781c8d731&p=13e255df270a4b81a6c7d21e384afe8a&pm=s

### CODE REVIEW

File: src/components/atoms/Input/index.tsx [hunk 1]
```diff
@@ -42,9 +53,10 @@ const Input: React.FC<InputProps> = ({
       )}
       <input
         className={cx(
-          'w-full rounded-lg bg-dark-100 text-gray-100 focus:outline-none h-[50px] pb-[4px] placeholder:text-gray-500',
+          'align-middle h-[46px] w-full rounded-lg bg-dark-100 text-gray-100 focus:outline-none placeholder:text-gray-500',
```
Comment (pedrohbl03) [inline src/components/atoms/Input/index.tsx]: add a new styles to change default style by google for autofill values. 

We can make this in tailwind 3+ using "autofill" pseudo-element.

look this exemple:

`"autofill:bg-transparent"`

you can check more about this in: [Tailwind Autofill Docs](https://tailwindcss.com/docs/hover-focus-and-other-states#autofill
)

File: src/pages/auth/login.tsx [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+    setPasswordInvalid(false);
+    setPassword(e.target.value);
+  };
+
+  const emailChange = (e: any) => {
+    setEmailInvalid(false);
+    setEmail(e.target.value);
+  };
+
+  const handleSubmit = (e: { preventDefault: () => void }) => {
```
Comment (pedrohbl03) [inline src/pages/auth/login.tsx]:
```suggestion
  const handleSubmit = (e: React.ChangeEvent<HTMLInputElement>) => {
```
We can make this ?

**Review by pedrohbl03 — CHANGES_REQUESTED** [review]

File: src/pages/auth/login.tsx (hunk 2 above)
Comment (femacedogomes) [reply src/pages/auth/login.tsx]: maded

**Review by renanpontez — APPROVED** [review]
Great work!

**Pull request merged**